cuBERT_topic_modelling/cuBERTopic.py

```diff
@@ -158,28 +159,23 @@ def extract_top_n_words_per_topic(
         """
 
         words = count.get_feature_names().to_arrow().to_pylist()
-
         labels = sorted(docs_per_topics_topics.to_arrow().to_pylist())
-
         indices = top_n_idx_sparse(tf_idf, n)
-        scores = top_n_values_sparse(tf_idf, indices)
-        sorted_indices = cp.argsort(scores, 1)
-        indices = cp.take_along_axis(indices, sorted_indices, axis=1)
-        scores = cp.take_along_axis(scores, sorted_indices, axis=1)
```

Comment:
got rid of all this code, realizing we don't need to do all this now, thanks for pointing it out @VibhuJawa! we can do it in much easier way now (`test_fit_transform` passes). Anything we needed to do from the way `BERTopic` implements it has been handled below.